server/utils/utils.ts

```diff
@@ -20,7 +23,12 @@ function convertToTitleCase(sentence: string | null): string {
   return isBlank(sentence) ? '' : sentence!.split(' ').map(properCaseName).join(' ')
 }
 
+function isInitialAssessmentAppointment(appointmentDetails: Appointment | ActionPlanAppointment): boolean {
```

Comment:
can we take this out of `sessionStatus.ts` now and import it from here instead?

Reply:
Yes. I forgot it was there! I think I do this a few times.

Seems that we need a general AppointmentUtils class?

Reply:
I think `AppointmentDecorator` would be a sensible place for it, I don't think we'd need to change the name

Reply:
Thanks I've done this